boards/components/src/round_robin.rs

```diff
@@ -0,0 +1,73 @@
+//! Component for a round robin scheduler.
```

Comment:
Should all of the scheduler components maybe go in another `scheduler` subdir/sub-crate?